fullstop-violation/fullstop-violation-persister-jpa/src/main/resources/db/migration/V14__add_columns_to_violations.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE fullstop_data.violation
```

Comment:
maybe we will a migration strategy here, e.g. parse app id and version from the meta data to initially populate those new fields